client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -54,10 +58,15 @@ const SiteName = styled.h2`
 `;
 
 const SiteUrl = styled.a`
+	display: flex;
+	align-items: center;
 	color: var( --studio-gray-60 );
 	&:visited {
 		color: var( --studio-gray-60 );
 	}
+	span {
```

Comment:
So, generally, in CSS-in-JS libraries, we try to avoid cascade effects because the components (should be) atomic, meaning they yield no side effects in their tree.

Imagine you're adding another `span` element as a child of `SiteUrl`. That element would inherit the same margin, even if you don't want it to happen...

In order to fix this, I would create a separate `span` element and apply the margin to it. Something like:

```js
const SiteUrlAddress = styled.span`
  margin-left: 4px;
`

<SiteUrlAddress>{ displaySiteUrl( site.URL ) }</SiteUrlAddress>
```

But thinking one step ahead, it makes more sense to style the `Icon` itself:

```js
const SiteStateIcon = styled(Icon)`
  margin-left: 4px;
`
```

Because the icon is the graphical element there and it's the one that it's purely cosmetic, despite having contextual meaning.

Reply:
> So, generally, in CSS-in-JS libraries, we try to avoid cascade effects because the components (should be) atomic, meaning they yield no side effects in their tree.

@zaguiini Ah, I didn't know this!

I'll apply some of your suggested changes tomorrow.